### ISSUE TITLE
ß doesn't working corectly in NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2490]
**Reported by Giovani on 2012-06-24 17:25**
Hi! A letter of German alphabet ß doesn't working correctly in NVDA translation other tham German. This is very bad for Me, because I using a Slovak NVDA and I would like to learn a German. Please, could You edit this think.
I am sorry for My terrible English.

Thank You, Marco Oros.

**Comment by dkager** [comment]
Closing because of a lack of feedback from the original reporter. This character (sharp 's', ß) is pronounced correctly if you arrow over it with the default espeak-ng langauge (english).
If the problem lies in reading this character as part of a word it would be nice to have a new issue with steps to reproduce.
